### PR TITLE
fix(tooltip): does not show tooltip when content is false

### PR DESCRIPTION
**Pull request by chenc041**
## Types of changes

- [ ] New feature
- [x] Bug fix
- [ ] Documentation change
- [ ] Coding style change
- [ ] Component style change
- [ ] Refactoring
- [ ] Performance improvement
- [ ] Test cases
- [ ] Continuous integration
- [ ] Typescript definition change
- [ ] Breaking change
- [ ] Others 

## Changelog

| Component | Changelog(CN) | Changelog(EN) | Related issues |
| --------- | ------------- | ------------- | -------------- |
|   Tooltip        |      `Tooltip` content 值是 false 时不显示弹窗         |     ` Tooltip ` content value is false does not display the popup          |       Close: [#571](https://github.com/arco-design/arco-design/issues/571)         |

## Checklist:

- [x] Test suite passes (`npm run test`)
- [ ] Provide changelog for relevant changes (e.g. bug fixes and new features) if applicable.
- [ ] Changes are submitted to the appropriate branch (e.g. features should be submitted to `feature` branch and others should be submitted to `main` branch)

### CODE REVIEW

File: components/Tooltip/index.tsx [hunk 1]
```diff
@@ -82,13 +82,7 @@ function Tooltip(baseProps: PropsWithChildren<TooltipProps>, ref) {
...
   // it fails in cases such as content = <>&nbsp;&nbsp;</>
   const isEmpty = (content: ReactNode): boolean => {
-    if (content === null || content === undefined) {
-      return true;
-    }
-    if (typeof content === 'string' && content.trim() === '') {
-      return true;
-    }
-    return false;
+    return !content || (typeof content === 'string' && content.trim() === '');
```
Comment (yinkaihui) [inline components/Tooltip/index.tsx:85]: content = 0  情况处理下 ？

Comment (chenc041) [reply components/Tooltip/index.tsx]: 怪我手贱....
现在好啦!
这个几个不展示的情况 测试用例也补齐了

File: components/Tooltip/__test__/index.test.tsx [hunk 2]
```diff
@@ -42,6 +42,32 @@ describe('Tooltip', () => {
...
+        </Tooltip>
+      );
+      expect((wrapper.find('Trigger').state() as TriggerState).popupVisible).toBe(false);
+      wrapper.update();
+      wrapper.find('Button').simulate('mouseenter');
+      jest.runAllTimers();
+      expect((wrapper.find('Trigger').state() as TriggerState).popupVisible).toBe(false);
+    });
+  });
+  it('does not show tooltip when content is 0', () => {
```
Comment (yinkaihui) [inline components/Tooltip/__test__/index.test.tsx]: 描述写反了～ 
另外写测试的时候可以尽量从 UI 上来写，直接读取组件的 state 的话，一旦组件调整为函数类型，state 是读取不到的。 而且也不能确定 UI 表现是否是我们所期望的。之前我们写测试的时候很多也是直接读的state，后续也没有调整，但新增的测试用例可以尽量避免一下哈～

Comment (chenc041) [reply components/Tooltip/__test__/index.test.tsx]: 不对呀 我提交完就发现写反了有提交了一个改掉了呀 你这个咋还是老的

测试用例这个 我下次注意, 因为也没有见到啥说明, 就按照之前的形式写的 😂

**Pull request merged**